tools/message-screenshot.js

```diff
@@ -0,0 +1,66 @@
+const puppeteer = require("puppeteer");
+const commander = require("commander");
+const path = require("path");
+const host = "localhost:9991";
+const options = {};
+
+commander
+    .arguments('<integration> <message_id>')
+    .action((integration, messageId) => {
+        options.integration = integration;
+        options.messageId = messageId;
+        console.log(`Capturing screenshot for ${integration} using message ${messageId}`);
+    })
+    .parse(process.argv);
+
+if (options.integration === undefined) {
+    console.error('no integration specified!');
+    process.exit(1);
+}
+
+async function run() {
+    const browser = await puppeteer.launch({
+        args: [
+            '--window-size=1400,1024',
+            '--no-sandbox', '--disable-setuid-sandbox',
+            '--font-render-hinting=none',
+        ],
+        defaultViewport: null,
+        headless: true,
```

Comment:
Maybe worth adding some comments on these parameters and where it's important that they're different from the normal puppeteer parameters we use?  

Also there's probably a refactor we could do to have most of this share code with `frontend_tests/puppeteer_tests/00-realm-creation.js` so the outer `try/catch/` block is shared, for example.